WooCommerce/Classes/ViewRelated/Products/FavoriteProducts/ProductListViewSection.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import struct Yosemite.Product
+
+struct ProductListViewSection {
+
+    enum ProductListViewSectionType {
+        case favourites
```

Comment:
Super nit: let's keep `favorite` naming for consistency across different places.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/12287/commits/836c3c48e269d1901196bf8d526dfb154e5d9e24